src/utils/Paginator/paginator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import styles from './paginator.module.css';
+
+const paginator = (props) => {
+  const {
+    totalUsersCount, currentPage, onPageChanged, pageSize,
+  } = props;
+  
+  const pagesCount = Math.ceil(totalUsersCount / pageSize);
+  const pages = [...Array(pagesCount).keys()];
+
+  const pagesMap = () => pages.map((page) => {
```

Comment:
No, I meant:
`const pageMapper = (page) => {...};`
`...`
`const pagesMap = pages.map(pageMapper);`

Reply:
what's the difference? execution order is the same

Reply:
done